spec/typhoeus/hydra/cacheable_spec.rb

```diff
@@ -53,6 +53,31 @@
           end
         end
       end
+
+      context "when cache is specified on a request" do
+        before { Typhoeus::Config.cache = false }
+
+        context "when cache is false" do
+          let(:non_cached_request) { Typhoeus::Request.new(base_url, {:method => :get, :cache => false}) }
+
+          it "finishes request" do
```

Comment:
I think this spec is not doing anything 🤔

```ruby
request = Typhoeus::Request.new("https://www.google.com", { method: :get, cache: false })
request.response
# => nil
```

So I'm not exactly sure what we are testing here.

Reply:
Not sure if I'm loving this but thoughts?

```ruby
context "when cache is specified on a request" do
  before { Typhoeus::Config.cache = false }

  context "when cache is false" do
    let(:non_cached_request) { Typhoeus::Request.new(base_url, {:method => :get, :cache => false}) }

    it "initiates an HTTP call" do
      allow(Typhoeus::EasyFactory).to receive(:new).with(non_cached_request, hydra).and_call_original
      hydra.add(non_cached_request)
    end
  end

  context "when cache is defined" do
    let(:cached_request) { Typhoeus::Request.new(base_url, {:method => :get, :cache => cache}) }

    before { cache.memory[cached_request] = response }

    it "uses the cache instead of making a new request" do
      expect(Typhoeus::EasyFactory).not_to receive(:new)
      hydra.add(cached_request)
    end
  end
end
```

Reply:
@PericlesTheo The snippet above works for me. However, this https://github.com/typhoeus/typhoeus/blob/master/spec/typhoeus/request/cacheable_spec.rb#L59-L83 is being tested differently which I think is fine as well. We can be more consistent but should probably be a different PR.

Reply:
I see, we should definitely make them consistent. Could you apply the suggestions in the spec above and in a separate PR I can make them consistent.

Reply:
Will do!